time/src/test/java/net/jqwik/time/api/DurationTests.java

```diff
@@ -0,0 +1,594 @@
+package net.jqwik.time.api;
+
+import java.time.*;
+import java.util.*;
+
+import net.jqwik.api.*;
+import net.jqwik.api.constraints.*;
+import net.jqwik.api.statistics.*;
+import net.jqwik.testing.*;
+import net.jqwik.time.api.arbitraries.*;
+import net.jqwik.time.internal.properties.arbitraries.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+import static net.jqwik.testing.ShrinkingSupport.*;
+import static net.jqwik.testing.TestingSupport.*;
+
+@Group
+class DurationTests {
+
+	@Provide
+	Arbitrary<Duration> durations() {
+		return Times.durations();
+	}
+
+	@Group
+	class SimpleArbitraries {
+
+		@Property
+		void validDurationIsGenerated(@ForAll("durations") Duration duration) {
+			assertThat(duration).isNotNull();
+		}
+
+		@Group
+		class WorstCases {
+
+			@Property
+			void worstCaseGeneration(
```

Comment:
Numbered test cases are an anti-pattern. Can the numbers be replaced by a description.